### DISCUSSION TITLE
your email system is just like a joke

### DISCUSSION THREAD

**Discussion by dashuigang** [wonderunit/storyboarder#2287] [General]
your email system is just like a joke,doesn't work at all.
there seems to be only one mail in your world , called gmail......

**Comment by audionerd** [comment]
i have no idea what this means lol. are you having trouble downloading the Storyboarder app from the website?

**Reply by dashuigang** [reply]
means your website just only gmail can pass

**Reply by The-Maize** [reply]
@dashuigang that is not factual, it seems to be a situational fault, have you got blockers installed on your system? like noscript, or adguard?i am able to download and use my german mail service just fine, i do not use gmail.

**Comment by ramAustralia** [comment]
gmail is censoring (blocking) emails, especially from outside the USA.   They are causing significant problems in Australia to media producers and environmental groups.   Please host some real email addresses on your wonderunit server.

**Comment by ramAustralia** [comment]
https://rwmalonemd.substack.com/p/google-has-usurped-democracy
